src/graphql/operations/users.ts

```diff
@@ -16,6 +16,8 @@ export default async function (parent, args) {
   const whereQuery = buildWhereQuery(fields, 'u', where);
   const queryStr = whereQuery.query;
   const params: any[] = whereQuery.params;
+  const ids = (where.id_in || [where.id]).filter((id: any) => id);
+  if (Object.keys(where).length > 1) ids.length = 0;
```

Comment:
Don't get this part

Reply:
If there are more than 1 filters, users in `leaderboard` table will be ignored, to give more importance to these filters, 

If either `id_in` or `id` exist in filters, we fetch users from both tables

Reply:
`ids` is an array. You're manipulating the `length` property of an array, and assigning it a new value ?

Reply:
Yes yes, this will make `ids = []` an empty array if there are other filters in `where`

Reply:
Did not know that way of clearing an array ... the wonder of JS